src/components/landing/pages/history/common/sync-status.scss

```
@@ -0,0 +1,10 @@
+.sync-icon {
+  .fa {
+    opacity: 0.2;
+    transition: opacity 0.2s ease-in-out, color 0.2s ease-in-out;
+  }
+
+  &:hover .fa{
```

Same as above (missing space)
```suggestion
  &:hover .fa {
```